frontend/src/fixtures/commonsFixtures.js

```diff
@@ -23,7 +29,18 @@ const commonsFixtures = {
             "endDate": "6/11/2021",
             "totalPlayers": 50,
             "cowPrice": 15,
+            "milkPrice": 4,
+            "startingBalance": 200,
+            "startingDate": "6/11/2021",
         }
+        //{
+        // "id": 1,
+        // "name": "Anika's Commons",
+        // "startingDate": 5,
+        // "endDate": "6/11/2021",
+        // "cowPrice": 50,
+        // "milkPrice": 15,
+        //}
```

Comment:
Why is this commented out code still in the file?

Reply:
deleted commented out lines